omnibus/package-scripts/updater-deb/postinst

```diff
@@ -132,9 +134,9 @@ fi
 chmod 750 ${UPDATER_HELPER}
 setcap cap_setuid+ep ${UPDATER_HELPER}
 
-$INSTALL_DIR/bin/updater/updater bootstrap -P datadog-agent
-# Bootstrap installs first agent with root ownership, overriding to dd-agent
-chown -R dd-agent:dd-agent ${PACKAGES_DIR}
+$INSTALL_DIR/bin/updater/updater bootstrap -P datadog-agent > "{LOG_DIR}/updater.log"
```

Comment:
Don't we also want `stderr` in that file?

Reply:
I didn't redirect stderr as if the bootstrap fails, the deb installation fails and stderr should be propagated back straight instead of going to a file.
I havn't verified the behaviour with a e2e test though

Reply:
Ok that seems like a fair reason to me :)

Reply:
on second thoughts I'll make the stdout redirect to the log file from the binary itself, to ensure it has the right ownership (dd-updater:dd-updater), relying just on the chmod you suggested above